.eslintrc.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const path = require('path');
+
+module.exports = {
+	env: {
+		browser: true,
+		es6: true,
+		jest: true,
+		node: true,
+	},
+	extends: 'liferay/react',
+	parser: '@typescript-eslint/parser',
+	parserOptions: {
+		ecmaFeatures: {
+			jsx: true,
+		},
+	},
+	plugins: ['@typescript-eslint'],
+	rules: {
+		'@typescript-eslint/array-type': ['error', 'generic'],
+		'@typescript-eslint/interface-name-prefix': ['error', 'always'],
+		'@typescript-eslint/no-unused-vars': 'error',
+		'no-unused-vars': 'off',
+		'notice/notice': [
+			'error',
+			{
+				templateFile: path.join(__dirname, 'copyright.js'),
```

Comment:
And here's why we "needed" to change from JSON to JS for this config file: for consistency with other repos, we pass an absolute path here. This repo is a simple case, so it it would also work with plain old "copyright.js", but I think it's worth matching what we do in liferay-portal. As a bonus, that means we can now lint the config file itself, which means we get ordering enforcement etc.